packages/suite-desktop/src-electron/index.d.ts

```diff
@@ -69,7 +69,7 @@ declare type WinBounds = {
 
 declare type UpdateSettings = {
     skipVersion: string;
-    preUpdateVersion?: string;
+    savedCurrentVersion?: string;
```

Comment:
This would be a good place for a comment explaining what does this field contains and hopefully preventing the confusion you described [here](https://github.com/trezor/trezor-suite/pull/3946#discussion_r664330242).

Something like
```suggestion
    /**
      * Duplicates and persists `app.getVersion()` most of the time except the first app start after an update.
      * In that case it's used to detect an update to display a success notification to the user.
      * Use `app.getVersion()` to get the current version of the app.
      */
    savedCurrentVersion?: string;
```